### ISSUE TITLE
How to stop e from autosuggesting \emp

### ISSUE THREAD

**Issue by userrand** [James-Yu/LaTeX-Workshop#4011]
> _Please fill the following fields with a star (*) and provide as much related information as possible._

## Pre-checks*
> _Please change the following `[ ]` to `[x]` for confirmation._
- [ *] The feature request has not been suggested in this repository.

## The Missed*
> _Is your feature request related to a problem? Please provide a clear and concise description of what the problem is._
- 

With the settings, snippets only trigger when i press \, @ , maybe { not sure, and the letter e which trigger \emph . I do not see suggestions for other letters, how can i prevent e from triggering \emph (I am willing to remove that snippet entirely)

` "editor.suggestOnTriggerCharacters": false,
    "editor.acceptSuggestionOnEnter": "off",
    "editor.wordBasedSuggestions": false,
    "editor.snippetSuggestions": "none",
    "editor.suggest.showWords": false,
    "editor.suggest.matchOnWordStartOnly": false,
    "editor.suggest.showKeywords": false,
    "editor.acceptSuggestionOnCommitCharacter": false,`

## The Solution*
> _Please provide a solution you would like to have._

## Anything Else?
> _Add any other context about the feature request below._
-

**Comment by James-Yu** [comment]
Sorry but I don't understand your request. Can you please elaborate more with maybe screenshots?

**Comment by userrand** [comment]
Thank you for your help and apologies for my previous message, the text is indeed confusing particularly with the disappearance of `\` from the rendered markdown. 

With the settings given in the previous message, auto suggestions mostly only occur when i want them to, that is, when I press the keys

- `\`
- @
- maybe it happens with } but i did not test it

However, when typing the `e` key, I get an auto suggestion for `\emph` even with the settings above, see attached image. This does not occur with other letters which leads me to wonder whether the implementation of the snippet for`\emph` in the code is somehow different than with any other snippet.


![image](https://github.com/James-Yu/LaTeX-Workshop/assets/114431098/d0890d1d-f36a-431d-bdcf-64eea5afa6bb)

**Comment by jlelong** [comment]
I cannot reproduce the behaviour you describe. The command `\emph` is not defined differently than others. 

Which version of LW do you use? I am surprised by your screenshot because the command `\emph` supplied by LW has type _Methods and Functions_, see https://code.visualstudio.com/docs/editor/intellisense whereas your completion has type _Properties and Attributes_.

**Comment by userrand** [comment]
@jlelong  I am using the latest version I think, that is, 9.13.4.

 I suppose i can just download the extension from the marketplace, look for wherever the snippet file is and remove that particular snippet.
 
 Not sure how to install it from there but I suppose will figure it out if i decide to stay with vscode (I am currently trying different options)

**Comment by jlelong** [comment]
Let me clarify my previous message. I am not convinced that the `emph` suggestion you see comes from LW because it does not have the correct type. Here is a screenshot of the `\emph` completion provided by LW 
![Capture d’écran 2023-09-13 à 09 28 16](https://github.com/James-Yu/LaTeX-Workshop/assets/2910140/d269d14a-b7ad-4758-946c-02c819ec6fdd)

**Comment by userrand** [comment]
Indeed, it seems it is coming from another snippet extension that i was using. I did not realize that it also had a an intellisense menu. My apologies and thank you for your help and the great work that has been done on this extension.
